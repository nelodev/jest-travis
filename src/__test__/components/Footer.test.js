import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('Testing <Footer />', () => {
  const wrapper = mount(<Footer />);
  test('Should render the component', () => {
    expect(wrapper.length).toEqual(1);
  });

  test('Render title of the component', () => {
    expect(wrapper.find('.Footer-title').text()).toEqual('Platzi Store');
  });
});

describe('Testing snapshots', () => {
  test('Check UI of the component', () => {
    const footer = create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  });
});
