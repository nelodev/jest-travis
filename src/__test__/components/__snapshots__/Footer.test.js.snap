// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing snapshots Check UI of the component 1`] = `
<div
  className="Footer"
>
  <p
    className="Footer-title"
  >
    Platzi Store
  </p>
  <p
    className="Footer-copy"
  >
    Todos los Izquierdos Reservados
  </p>
</div>
`;
